01_lecture/homework/task0.js

```diff
@@ -0,0 +1,4 @@
+function isAnagram(str1, str2) {
```

Comment:
You need to start to get used to use TS typization
```suggestion
function isAnagram(str1: string, str2: string): boolean {
```

Reply:
Done for every task